### DISCUSSION TITLE
Reconsider the level number quest

### DISCUSSION THREAD

**Discussion by andrewharvey** [streetcomplete/StreetComplete#4856] [General]
I'm not convinced the "On which level number is this place located?" quest is suitable for SC. The [`level=*`](https://wiki.openstreetmap.org/wiki/Key:level) tag is the numeric floor number relative to ground, but for large shopping complexes, airports, train stations it can be hard to determine what level you're on relative to "ground" and more so when there multiple exits to the street level but from different levels.

The tag is distinct from `level:ref` which is much more suited for SC since it's directly surveyable from wayfinding signage, lift buttons etc.

What's worse is that wayfinding signage might say level 1 which you'd enter for `level:ref` but due to level starting from the lowest ground level in OSM it might be incorrect to set `level=1`.

It's best to therefore only ask for `level:ref` and leave `level` tagging to an editor where someone can look at the complete indoor mapping data, asses the whole site to determine where `level=0` is and ensure all the `level` tags are consistent within the site.

The only situation where asking for `level` would work is where we have a full indoor map showing like Google Maps where you can toggle between levels on the map and see how all the other shops are mapped.

**Comment by mnalis** [comment]
And then there is also [addr:floor=*](https://wiki.openstreetmap.org/wiki/Key:addr:floor) (which also happens to explain that [level:ref=*](https://wiki.openstreetmap.org/wiki/Key:level:ref) come hugely from mass-imports)

**Reply by andrewharvey** [reply]
`addr:floor` is good when it's used as part of addressing, but not a general solution, eg. a toilet within a shopping centre won't have an address.

Usage spikes may have come from imports, but it doesn't change the fact that neither `level` or `addr:floor` tags are suitable for tagging the level a feature is on within a building in a way that relates to wayfinding signage.
